edd-slack.php

```diff
@@ -620,17 +620,18 @@ public function get_notification_fields( $query = true ) {
 					'posts_per_page' => 99999
 				) );
 
-				foreach ( $discount_codes as $discount_code ) {
-
-					if ( $discount_code instanceof EDD_Discount ) {
-						$name = $discount_code->name;
-						$code = $discount_code->code;
-					} else {
-						$name = $discount_code->post_title;
-						$code = get_post_meta( $discount_code->ID, '_edd_discount_code', true );
+				if ( $discount_codes ) {
```

Comment:
```suggestion
				if ( $discount_codes && is_array( $discount_codes ) ) {
```

This extra `is_array()` check probably isn't necessary, but better to be safe than sorry! :joy: 

Thanks @mihaijoldis

Reply:
Agreed. Made the change